mne_bids/tests/test_read.py

```diff
@@ -706,3 +707,31 @@ def test_bads_reading():
         raw = read_raw_bids(bids_path=bids_path, verbose=False)
     assert type(raw.info['bads']) is list
     assert set(raw.info['bads']) == set(bads_bids)
+
+
+@pytest.mark.filterwarnings(warning_str['channel_unit_changed'])
+def test_write_read_fif_split_file():
+    """Test split files are read correctly."""
+    bids_root = _TempDir()
+    tmp_dir = _TempDir()
+    bids_path = _bids_path.copy().update(root=bids_root, datatype='meg')
+    raw = _read_raw_fif(raw_fname, verbose=False)
+    n_channels = len(raw.ch_names)
+    n_times = int(2.2e9 / (n_channels * 4))  # enough to produce a split
```

Comment:
sorry I am late to the party. Instead of creating a huge file during testing, I wonder if it would have made sense to artificially create a split file with the parameters of `raw.save` ?